tactics/docker.py

```diff
@@ -0,0 +1,28 @@
+from charmtools.build.tactics import WheelhouseTactic
+from charmtools.utils import Process
+import tempfile
+from path import Path as path
+
+
+class DockerWheelhouseTactic(WheelhouseTactic):
+    ''' The references to process, path, and tempfile are implicit
+    below because  they come from the global scope of
+    charmtools.utils which gets passed in during
+    charmtools.utils.load_class
+    Investigate there if there is abrupt breakage here.
+    '''
+    def __call__(self, venv=None):
+        create_venv = venv is None
+        venv = venv or path(tempfile.mkdtemp())
+        pip = venv / 'bin' / 'pip3'
+        wheelhouse = self.target.directory / 'wheelhouse'
+        wheelhouse.mkdir_p()
+        if create_venv:
+            Process(('virtualenv', '--python', 'python3', venv)).exit_on_error()()  # noqa
```

Comment:
That's just how the API of https://github.com/juju/charm-tools/blob/master/charmtools/utils.py#L167 works.  `Process()` creates an instance of the `Process` class, and `Process().exit_on_error()` returns `self`, so you then need to invoke the `__call__` method with the extra parens.  It could also be:

```python
Process('virtualenv', '--python', 'python3', venv), exit=True)()
```

It might be nicer if it supported a `run` method instead / also, to make it more clear that it's being executed.